app/src/main/kotlin/org/koitharu/kotatsu/reader/ui/pager/webtoon/WebtoonRecyclerView.kt

```diff
@@ -12,6 +14,17 @@ class WebtoonRecyclerView @JvmOverloads constructor(
 ) : RecyclerView(context, attrs, defStyleAttr) {
 
 	private var onPageScrollListeners: MutableList<OnPageScrollListener>? = null
+	private val detachedViews = ArrayList<View>()
```

Comment:
Is it possible to use `WeakHashMap` here? (Like in the `ActivityRecreationHandle`)